ui-charts/src/spaceTimeChart/components/OccupancyBlockLayer.tsx

```diff
@@ -15,6 +15,29 @@ export type OccupancyBlockLayerProps = {
   occupancyBlocks: OccupancyBlock[];
 };
 
+const getFillStyle = (ctx: CanvasRenderingContext2D, color: string) => {
+  if (color === 'blinking') {
+    const patternCanvas = document.createElement('canvas');
+    patternCanvas.width = 8;
+    patternCanvas.height = 8;
+
+    const pctx = patternCanvas.getContext('2d')!;
+
+    pctx.clearRect(0, 0, 8, 8);
+
+    pctx.strokeStyle = '#FFEABF';
+    pctx.lineWidth = 2;
+    pctx.beginPath();
+    pctx.moveTo(0, 8);
+    pctx.lineTo(8, 0);
+    pctx.stroke();
+
+    return ctx.createPattern(patternCanvas, 'repeat')!;
+  }
+
+  return color;
+};
+
```

Comment:
We have a mock-up for this pattern ?

Reply:
```suggestion
const getFillStyle = (ctx: CanvasRenderingContext2D, color: string, isBlinking?: boolean) => {
  if (isBlinking) {
    const patternCanvas = document.createElement('canvas');
    patternCanvas.width = 8;
    patternCanvas.height = 8;

    const pctx = patternCanvas.getContext('2d')!;

    pctx.clearRect(0, 0, 8, 8);

    pctx.strokeStyle = color;
    pctx.lineWidth = 2;
    pctx.beginPath();
    pctx.moveTo(0, 8);
    pctx.lineTo(8, 0);
    pctx.stroke();

    return ctx.createPattern(patternCanvas, 'repeat')!;
  }

  return color;
};

```
I propose this suggestion because we might also want to have other blinking signals (blinking red, blinking green).

Reply:
```suggestion
const getFillStyle = (ctx: CanvasRenderingContext2D, color: string, isBlinking?: boolean) => {
  if (isBlinking) {
    const patternCanvas = document.createElement('canvas');
    patternCanvas.width = 8;
    patternCanvas.height = 8;

    const pctx = patternCanvas.getContext('2d')!;

    pctx.clearRect(0, 0, 8, 8);

    pctx.strokeStyle = color;
    pctx.lineWidth = 2;
    pctx.beginPath();
    pctx.moveTo(0, 8);
    pctx.lineTo(8, 0);
    pctx.stroke();

    return ctx.createPattern(patternCanvas, 'repeat')!;
  }

  return color;
};

```
I propose this suggestion because we might also want to have other blinking signals (blinking red).

Reply:
> We have a mock-up for this pattern ?

No, unfortunately we only have 3 colors : https://www.sketch.com/s/bcbac4ff-7f74-4f1a-9e10-08d115c47ed8/p/C0FA571F-D55C-40A1-B0E0-B61D135726CD/canvas

Reply:
fixed !